README.md

```diff
@@ -46,7 +46,7 @@ Welcome to the **Interneers Lab 2025** repository! This serves as a minimal star
 2. **Configure Git** with your name and email:
    ```bash
    git config --global user.name "Your Name"
-   git config --global user.email "your.email@example.com"
+   git config --global user.email "your.email@example.com" # Use the same email you shared during onboarding
```

Comment:
I know we have already added the message in the top section however I think it should also be inlined where the message actually matters. 

<img width="694" alt="Screenshot 2025-02-17 at 6 56 38 PM" src="https://github.com/user-attachments/assets/28c66bba-85e6-4bf4-9d33-741faac40e96" />